### ISSUE TITLE
Use vue-router to handle Spotify auth callback

### ISSUE THREAD

**Issue by eamonnbell** [eamonnbell/sortzzi#7]
Currently, after auth, Spotify redirects to the application root with auth token appended as URI hash fragment. The token is extracted from the URI and tidied up using Web APIs. 

Application should supply an auth route to handle the token using the vue-router APIs for path manipulation and return to the application root once authenticated. 

The authentication route should also be able to initiate a login. Currently, the login URI is manually constructed and used as the href in an anchor tag.

**Comment by betoflakes** [comment]
Hi there, have found a solution for this?

The same issue here LOL

**Comment by eamonnbell** [comment]
No, not yet - I don't have time to work on this very often, sorry.

**Comment by betoflakes** [comment]
Same here
